clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemCore.java

```diff
@@ -182,6 +183,17 @@ boolean rename(TachyonFile src, TachyonURI dst, RenameOptions options) throws IO
       FileDoesNotExistException, TachyonException;
 
   /**
+   * Sets the acl of a file or directory.
+   *
+   * @param path to be set acl on
+   * @param options the acl option to be set
+   * @throws TachyonException if a Tachyon error occurs
+   * @throws IOException an I/O error occurs
+   */
+  public void setAcl(TachyonURI path, SetAclOptions options) throws TachyonException,
+      IOException;
```

Comment:
Could `Acl` be a part of the file's `State` and use the `setState` api to modify?